README.md

```diff
@@ -26,12 +26,12 @@ The high level features of this ManageIQ extension.
 * Dynamic dialog methods for populating drop down fields with LDAP entries using specified attributes as the value and description
 
 # Dependencies
-Dependencies of this ManageIQ extensions.
+Dependencies of this ManageIQ extension.
 
 ## Other Datastores
 These ManageIQ atuomate domains must also be installed for this datastore to function.
 
-* [RedHatConsulting_Utilities](https://github.com/rhtconsulting/miq-Utilities)
+* [RedHatConsulting_Utilities](https://github.com/RedHatOfficial/miq-Utilities)
```

Comment:
Should we update the title section of this line to reflect something other than the old RedHatConsulting?

Reply:
Let's do this later since we need to discuss the new name and change it in many places.